go/config/config_test.go

```diff
@@ -244,23 +244,19 @@ func TestConfig_AzAffinity(t *testing.T) {
 }
 
 func TestConfig_InvalidRequestAndConnectionTimeouts(t *testing.T) {
-	var errorType *errors.ConfigurationError
```

Comment:
We probably need this, it isn't a default error type

Reply:
In order to have this test use a custom error type, it needs to move to an integration test.

Reply:
It does not require a server to test this.